lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -13,7 +13,13 @@ module CreditCardMethods
         'maestro'            => ->(num) { (12..19).include?(num&.size) && in_bin_range?(num.slice(0, 6), MAESTRO_RANGES) },
         'forbrugsforeningen' => ->(num) { num =~ /^600722\d{10}$/ },
         'sodexo'             => ->(num) { num =~ /^(606071|603389|606070|606069|606068|600818)\d{8}$/ },
-        'vr'                 => ->(num) { num =~ /^(627416|637036)\d{8}$/ }
+        'vr'                 => ->(num) { num =~ /^(627416|637036)\d{8}$/ },
+        'carnet'             => lambda { |num|
```

Comment:
I take it you're not going with the stabby lambda `->(num)` here for readability sake?

Reply:
It's actually a RuboCop violation to use `->` with a multi-line lambda. Was news to me, too.